easybuild/easyconfigs/b/Bowtie2/Bowtie2-2.4.1-GCC-9.2.0.eb

```diff
@@ -0,0 +1,39 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# Modified by: Robert Schmidt <rjeschmi@gmail.com>
+# Ottawa Hospital Research Institute - Bioinformatics Team
+# Modified by: Adam Huffman <adam.huffman@crick.ac.uk>
+# The Francis Crick Institute
+# Modified by: Kurt Lust, UAntwerp
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+name = 'Bowtie2'
+version = '2.4.1'
+
+homepage = 'http://bowtie-bio.sourceforge.net/bowtie2/index.shtml'
+description = """ Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads
+ to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s
+ of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes.
+ Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome,
+ its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes."""
+
+toolchain = {'name': 'GCC', 'version': '9.2.0'}
```

Comment:
@sassy-crick why don't you use GCC-9.3.0, the version included in foss-2020a?

Reply:
@smoors : I wanted to but neither foss-2020a nor GCC-9.3.0 are yet in the latest official release which is EB-4.2.0. There is only GCCcore-9.3.0 in that release. So I reasoned there might be a problem with this and I decided to go for cutting edge. I hope that helps.

Reply:
@sassy-crick Yeah so, when you are at the cutting edge, one will have to sync with `develop` a lot. 9.2.0 is not a great choice, as it won't be part of any toolchain. By the next EB release, there will already be a Bowtie2 at GCC-9.3.0, so, I'm afraid there isn't much use for this version

Reply:
No worries. Is there a way to check for me quickly if what I want to submit as a PR has already a PR? I know the develop branch but that only has stuff in it which has been accepted. 
Happy to close this PR, so no worries.

Reply:
Check develop branch and search current PR's is the only way i know of. Not a big problem if one misses it, that's why us maintainers review all PRs to see what goes into the next release.

Reply:
Thanks. I will close the PR.